File: Implemetation, Math/Python/4948 베르트랑 공준.py
from sys import stdin

def isPrime(number) :
    if number % 2 == 0 : return False;
    for i in range(3, int(number ** 0.5)+1, 2) :
        if number % i == 0 : return False;
    return True

while True :
    n = int(stdin.readline())
    if n == 0 : break
    if n == 1 : 
        print(1); continue
    count = 0
    for i in range(n+1, (n*2) + 1) :
        if isPrime(i) :
            count += 1
    
    print(count)
    

File: Beakjoon/_etc/C#/4948 베르트랑 공준.cs
using System;
using static System.Console;

class Program
{
    static bool isPrime(int number) {
        if(number % 2 == 0) return false;
        for(int i = 3; i <= (int)Math.Sqrt(number); i+=2) {
            if(number % i == 0) return false;
        }
        return true;
    }
    static void Main() {
        while(true) {
            string s = ReadLine();
            if(s[0] == '0') break;
            int n = int.Parse(s);
            if(n == 1) {
                WriteLine(1); continue;
            }
            int count = 0;
            for(int i = n+1; i <= n*2; i++) {
                if(isPrime(i)) count ++;
            }
            
            WriteLine(count);
        }
    }
}
